File: app/src/main/java/se/appshack/android/refactoring/main/data/model/PokemonSpeciesResponse.java
package se.appshack.android.refactoring.main.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PokemonSpeciesResponse {

    @SerializedName("id")
    public int id;

    @SerializedName("name")
    public String name;

    @SerializedName("genera")
    public List<GenusResponseModel> genera;
}

File: app/src/main/java/se/appshack/android/refactoring/main/data/model/PokemonSpeciesResponse.kt
package se.appshack.android.refactoring.main.data.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class PokemonSpeciesResponse (

    @SerializedName("id")
    var id: Int = 0,

    @SerializedName("name")
    var name: String? = null,

    @SerializedName("genera")
    var genera: List<GenusResponseModel>? = null

): Parcelable

